### PR TITLE
chore: add the ActiveColorDark setting

### PR DESCRIPTION
**Pull request by FeiWang1119**
get the active color according to system theme(light or dark). 
There are two different active colors between light and dark theme.

Log:

### CODE REVIEW

File: src/kernel/dguiapplicationhelper.cpp [hunk 1]
```diff
@@ -1049,7 +1059,16 @@ DPalette DGuiApplicationHelper::fetchPalette(const DPlatformTheme *theme)
 
     bool ok = false;
     base_palette = theme->fetchPalette(standardPalette(type), &ok);
-    const QColor &active_color = theme->activeColor();
+    QColor activeColor;
```
Comment (18202781743) [inline src/kernel/dguiapplicationhelper.cpp:1062]: 这个fallback的逻辑封装一下，

File: src/kernel/dguiapplicationhelper.cpp [hunk 2]
```diff
@@ -345,10 +346,19 @@ void DGuiApplicationHelperPrivate::_q_initApplicationTheme(bool notifyChange)
...
     QObject::connect(appTheme, &DPlatformTheme::paletteChanged, app, onAppThemeChanged);
-    QObject::connect(appTheme, &DPlatformTheme::activeColorChanged, app, [this] {
+    QTimer *timer = new QTimer(appTheme);
+    timer->setInterval(100);
+    timer->setSingleShot(true);
+    QObject::connect(timer, &QTimer::timeout, timer, [this] {
         if (!appPalette)
             notifyAppThemeChanged();
     });
+    QObject::connect(appTheme, &DPlatformTheme::activeColorChanged, app, [timer]{
```
Comment (18202781743) [inline src/kernel/dguiapplicationhelper.cpp:356]: 直接捕获，timer可能为野指针，

File: include/kernel/dplatformtheme.h [hunk 3]
```diff
@@ -35,6 +35,7 @@ class DPlatformTheme : public DNativeSettings
     Q_PROPERTY(QByteArray gtkFontName READ gtkFontName WRITE setGtkFontName NOTIFY gtkFontNameChanged)
 
     Q_PROPERTY(QColor activeColor READ activeColor WRITE setActiveColor NOTIFY activeColorChanged)
+    Q_PROPERTY(QColor activeColorDark READ activeColorDark WRITE setActiveColorDark NOTIFY activeColorDarkChanged)
```
Comment (18202781743) [inline include/kernel/dplatformtheme.h:38]: 名称是不是应该叫darkActiveColor？

File: src/kernel/dguiapplicationhelper.cpp (hunk 1 above)
Comment (FeiWang1119) [reply src/kernel/dguiapplicationhelper.cpp:1062]: done

**Review by 18202781743 — APPROVED** [review]

**Pull request merged**